src/components/Table/index.js

```diff
@@ -67,4 +68,8 @@ const TableRefForwarded = forwardRef((props, ref) =>
   <Table ref={ref} {...props} />
 );
 
+Table.defaultProps = {
+  hiddenColumns: []
+}
```

Comment:
Pikkunillitys.

Tän legacyhtävän defaultProps-atribuutin sijaan voi nykyään käyttä ES6 destructuring default arvoja suoraan funktiomäärittelyssä: https://dev.to/prototyp/javascript-es6-destructuring-with-default-values-2aj6

Reply:
Päätin korjata